File: backend/src/main/java/com/example/Flicktionary/domain/series/repository/SeriesCastRepository.java
package com.example.Flicktionary.domain.series.repository;

import com.example.Flicktionary.domain.series.entity.SeriesCast;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface SeriesCastRepository extends JpaRepository<SeriesCast, Long> {

    List<SeriesCast> findSeriesByActorId(Long actorId);
}

File: backend/src/main/kotlin/com/example/Flicktionary/domain/series/repository/SeriesCastRepository.kt
package com.example.Flicktionary.domain.series.repository

import com.example.Flicktionary.domain.series.entity.SeriesCast
import org.springframework.data.jpa.repository.JpaRepository

interface SeriesCastRepository : JpaRepository<SeriesCast, Long> {
    fun findSeriesByActorId(actorId: Long): List<SeriesCast>
}
